src/orion/core/__init__.py

```diff
@@ -23,6 +23,7 @@
 __version__ = VERSIONS['version']
 __license__ = 'BSD-3-Clause'
 __author__ = u'Epistímio Team'
+__author_short__ = u'Epistímio Team'
```

Comment:
@bouthilx what is the need for this? Shall we name it without the ` Team` in front?

Reply:
I named it `Epistímio Team` because it was `Oríon Team`, but I do prefer `Epistímio` indeed.

Reply:
Same here, I preter `Epistímio`